labstats/bin/init.py

```diff
@@ -55,12 +55,15 @@
     SELECT `user`, `host`, `start` FROM `session` WHERE `end` IS NULL;
 
 CREATE VIEW `users_in_lab_count_public` AS
-    SELECT count(*) as `count` FROM `users_in_lab`;
+    SELECT COUNT(DISTINCT `user`) AS `count` FROM `users_in_lab`;
 
 CREATE VIEW `staff_in_lab_public` AS
     SELECT `user`, `host`, `start` FROM `users_in_lab` WHERE `user` IN (
         SELECT `user` FROM `staff`
     );
+    
+CREATE VIEW `staff_in_lab_count_public` AS
+    SELECT COUNT(DISTINCT `user`) AS `count` FROM `staff_in_lab_public`; 
```

Comment:
Originally, I thought it would be easier to just dump ocflib.lab.stats.staff_in_lab into a set in ocfweb/stats/summary.py and return its len(), but after reading the summary.html template and going back through ocflib to see where the data was coming from IMHO it seems adding the view is a more correct solution. I still have the draft code for the other version, I'll submit that as a PR so you can take a look at what my thought process was.